less/modals.less

```diff
@@ -62,10 +62,9 @@
 
 // Modal background
 .modal-backdrop {
-  position: fixed;
+  position: absolute;
   top: 0;
   right: 0;
-  bottom: 0;
```

Comment:
+1

Removing `bottom: 0;` causes the calculated height of `.modal-backdrop` to be 0.